c2corg_ui/static/js/documentediting.js

```diff
@@ -80,9 +81,9 @@ app.DocumentEditingController = function($scope, $element, $attrs,
 
   /**
    * @type {app.Authentication}
-   * @private
+   * @export
```

Comment:
Do all these properties have to be exported? Or do you just want that they can be accessed from the child controllers?